### ISSUE TITLE
Dart 5.1 simwindow non consistent frame rate, glutTimerFunc is called more than once.

### ISSUE THREAD

**Issue by JiaYeLi** [dartsim/dart#729]
If I press "space key" twice, the frame time rate will increase.
And simulation window display discontinuous results.

so that I add some codes in MyWindow::keyboard() near where "glutTimerFunc" is called.
```cpp
//initilization firstPressPlay = true
case ' ':
      mSimulating = !mSimulating;
      if (mSimulating) {
        mPlay = false;
        if(firstPressPlay)   //added line
            glutTimerFunc(mDisplayTimeout, refreshTimer, 0);
        firstPressPlay = false;   //added line
      }
      break;
```

Is there a better way to solve this problem?

**Comment by tianyang9310** [comment]
I have come across the same problem. Is there any official fix?

**Comment by jslee02** [comment]
This should be resolved by #794.

**Comment by jslee02** [comment]
Closing. Reopen if it still doesn't work.
